commands/instances.go

```diff
@@ -842,29 +842,30 @@ func Upgrade(ctx context.Context, req *rpc.UpgradeRequest, downloadCB DownloadPr
 
 // LoadSketch collects and returns all files composing a sketch
 func LoadSketch(ctx context.Context, req *rpc.LoadSketchRequest) (*rpc.LoadSketchResponse, error) {
-	sketch, err := builder.SketchLoad(req.SketchPath, "")
+	// TODO: This a ToRpc function for the Sketch struct
```

Comment:
The meaning of the "This a ToRpc function for the Sketch struct" comment isn't clear to me.

Reply:
There's a `should be` missing between `This` and `a`, I'll update the comment.